app/javascript/components/editor/TestSummary.tsx

```diff
@@ -44,7 +50,9 @@ export function TestSummary({
         {isPresent(test.testCode) ? (
           <div className="--info">
             <h3>Code Run</h3>
-            <pre>{test.testCode}</pre>
+            <pre ref={testCodeRef}>
+              <code className="ruby">{test.testCode}</code>
```

Comment:
@kntsoriano This needs to be the `track.highlightjsLanguage` - could you proxy the info here pls?